packages/experimental/RadioGroup/src/Radio/Radio.types.ts

```diff
@@ -109,6 +124,18 @@ export interface RadioTokens extends FontTokens, IColorTokens, IForegroundColorT
    */
   subtextMarginBottom?: ViewStyle['marginBottom'];
 
+  /**
+   * Ripple color for Android.
+   * @platform android
+   */
+  rippleColor?: ColorValue;
+
+  /**
+   * Ripple radius for circular radio on Android.
+   * @platform android
```

Comment:
Can you clarify the usage need for `rippleColor` and `rippleRadius` tokens? Why did we need them?

Reply:
Added more info, updated checkbox and button ripple documentation as well